src/Tribute.js

```diff
@@ -261,6 +261,8 @@ class Tribute {
     let wrapper = this.range.getDocument().createElement("div"),
       ul = this.range.getDocument().createElement("ul");
     wrapper.className = containerClass;
+    wrapper.setAttribute('aria-expanded','true');
+    ul.setAttribute('id','results');
```

Comment:
Maybe this should be more specific to tribute? Maybe `tributeResults`.

Reply:
Yes, you're right. Change done !